src/main/scala/io/iohk/ethereum/jsonrpc/EthService.scala

```diff
@@ -924,4 +937,13 @@ class EthService(
     getTransactionsFromPool.map { resp =>
       Right(EthPendingTransactionsResponse(resp.pendingTransactions))
     }
+
+  /**
+    * Returns the account- and storage-values of the specified account including the Merkle-proof.
+    */
+  def getProof(req: GetProofRequest): ServiceResponse[GetProofResponse] = {
```

Comment:
so for every request sent - the new service is created? even for the requests with the same data?

Reply:
Taken care of here. It is slightly bigger change: https://github.com/input-output-hk/mantis/pull/869/commits/e2f01c4e4736e90bef9a4c203ee7c24fb56bf2a7